src/AssociationRegistry.Admin.Api/Verenigingen/Stop/RequestModels/StopVerenigingRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace AssociationRegistry.Admin.Api.Verenigingen.Stop.RequestModels;
+
+using Acties.StopVereniging;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using Vereniging;
+
+[DataContract]
+public class StopVerenigingRequest
+{
+    /// <summary>
+    /// De datum waarop de vereniging gestopt werd.
+    /// </summary>
+    [DataMember]
+    [Required]
+    public DateOnly? Einddatum { get; set; }
```

Comment:
This field is required and should never be null.

Reply:
Reden om dit toch nullable te maken was:
 - Als de string niet can geparsed worden naar een geldige DateOnly, dan krijgt de property op zijn default waarde. Bij Dateonly is dit DateOnly.MinValue (01/01/0001), bij DateOnly? is dit null

Reply:
Alright, good point